### ISSUE TITLE
Crash after upgrade to 1.4

### ISSUE THREAD

**Issue by adaszek** [sphinx-contrib/confluencebuilder#438]
Hi,

Thanks for adding support for :numref: in #422. 
The workaround with :math: allowed me to generate confluence pages properly. Unfortunately after upgrade i have a new unrelated exception:

`Exception occurred:
  File "/home/adaszyns/.local/lib/python3.7/site-packages/docutils/nodes.py", line 2032, in unknown_departure
    % (self.__class__, node.__class__.__name__))
NotImplementedError: <class 'sphinxcontrib.confluencebuilder.translator.storage.ConfluenceStorageFormatTranslator'> departing unknown node type: plantuml
The full traceback has been saved in /tmp/sphinx-err-4ga95706.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!`

Log file: https://gist.github.com/adaszek/c98f748e4442993ec75689f77e4164ec

**Comment by jdknight** [comment]
@adaszek, the changes introduced in v1.4 resulted in some breaking changes for sphinx-contrib plantuml's integration of this extension. I have [submitted a pull request](https://github.com/sphinx-contrib/plantuml/pull/51) to that project to help support the new changes.

As a workaround, you can inject the following in your configuration to help process `plantuml` nodes to work with this extension:

```
from docutils import nodes
from sphinxcontrib.plantuml import _prepare_html_render as plantuml_preparerender
from sphinxcontrib.plantuml import _render_batches_on_vist as plantuml_visitrender
from sphinxcontrib.plantuml import plantuml as plantuml_node
from sphinxcontrib.plantuml import render_plantuml as plantuml_render

def confluence_visit_plantuml(self, node):
    plantuml_visitrender(self)
    fmt = self.builder.config.plantuml_output_format
    with plantuml_preparerender(self, fmt) as (fileformats, _):
        _, outfname = plantuml_render(self, node, fileformats[0])

    # put node representing rendered image
    img_node = nodes.image(uri=outfname, alt=node.get('alt', node['uml']))
    node.replace_self(img_node)
    self.visit_image(img_node)

def confluence_depart_plantuml(self, node):
    pass

def setup(app):
    app.add_node(plantuml_node,
        confluence=(confluence_visit_plantuml, confluence_depart_plantuml),
        singleconfluence=(confluence_visit_plantuml, confluence_depart_plantuml),
        override=True)
```

Note though that if you are using the `singleconfluence` builder, you will need to use the most recent development implementation as a fix needed to support injected images for the `singleconfluence` builder was addressed in #451.

**Comment by jdknight** [comment]
The above changes are now available in sphinxcontrib-plantuml's v0.20 release. The use of plantuml should now work again for this extension when using the `confluence` builder when using the v1.4 release.
